site/en/gemma/docs/paligemma/inference-with-keras.ipynb

```diff
@@ -1,43 +1,16 @@
 {
   "cells": [
-    {
```

Comment:
the block containing the copyright looks like it's getting lost in the change and should be preserved